src/proxy.cpp

```diff
@@ -66,6 +66,8 @@
     delete poller_receive_blocked;\
     delete poller_send_blocked;\
     delete poller_both_blocked;\
+    delete poller_frontend_only;\
+    delete poller_backend_only\
```

Comment:
@bluca `CHECK_RC_EXIT_ON_FAILURE()` has an extra ";" after curly braces when called.
Maybe I should fix that also, removing ";" when called or changing macro so ";" at the end is syntactically correct?